extension.meta.xml

```diff
@@ -14,6 +14,9 @@
 		</author>
 	</authors>
 	<releases>
+		<release version="1.7.2" date="2017-05-19" min="2.5.x" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
```

Comment:
Where did you get `php-min` and `php-max` from? As far as I can tell, it's not part of the schema, or at least it’s not documented? http://symphonyextensions.com/schemas/extension/1.0/#changelogs-and-compatibility